xbmc/guilib/GUIControlGroup.cpp

```diff
@@ -408,23 +383,17 @@ void CGUIControlGroup::UnfocusFromPoint(const CPoint &point)
   CPoint controlCoords(point);
   m_transform.InverseTransformPosition(controlCoords.x, controlCoords.y);
   controlCoords -= GetPosition();
-  for (iControls it = m_children.begin(); it != m_children.end(); ++it)
-  {
-    CGUIControl *child = *it;
+  for (auto *child : m_children)
     child->UnfocusFromPoint(controlCoords);
-  }
   CGUIControl::UnfocusFromPoint(point);
 }
 
 bool CGUIControlGroup::HasID(int id) const
 {
   if (CGUIControl::HasID(id)) return true;
-  for (ciControls it = m_children.begin(); it != m_children.end(); ++it)
-  {
-    CGUIControl *child = *it;
+  for (auto *child : m_children)
```

Comment:
in multi-line fors (even if its just a single if statement) I prefer to leave the {}, but no biggie